core/src/test/java/io/parsingdata/metal/AutoEqualityTest.java

```diff
@@ -437,4 +436,11 @@ public void basicNoHashCollisions(final Object object, final Object same, final
         }
     }
 
+    private static void assertEquals(final Object o, final Object object) {
```

Comment:
Decided to override the assertEquals methods, so we get more meaningful error messages when a test fails. You will now see which objects causes a test to fail, which previously could only be discovered by debugging the test, which takes much more time.